spec/support/shared_examples/behaves_like_invitable.rb

```diff
@@ -0,0 +1,95 @@
+# shared examples for Invitable
+# Example usage:
+#  include_examples "Invitable", :workship_invitation, :workshop
+#
```

Comment:
This looks great, but can we remove the comment please. I think documentation about shared examples is easily available and we don't need comments in the code.

Reply:
done!